### PR TITLE
feat: bump staking withdraw gas limit for HW users

### PR DESCRIPTION
**Pull request by soulBit**
https://taiga.sovryn.app/project/sovryn-light/issue/653

### CODE REVIEW

**Review by tiltom — DISMISSED** [review]

File: src/app/pages/LandingPage/components/Promotions/index.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const Promotions: React.FC = () => {
             duration="Ongoing weekly rewards"
```
Comment (bitcoinuser) [inline src/app/pages/LandingPage/components/Promotions/index.tsx:23]: Hi @soulBit, please add this strings to be possible to translate to PT-BR

File: src/app/pages/LandingPage/components/Promotions/index.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ export const Promotions: React.FC = () => {
...
             text="Provide any amount of XUSD to the XUSD lending pool and instantly start accruing your share of 15,000 SOV rewards."
```
Comment (bitcoinuser) [inline src/app/pages/LandingPage/components/Promotions/index.tsx:24]: Hi @soulBit, please add this strings to be possible to translate to PT-BR

File: src/app/pages/LandingPage/components/Promotions/index.tsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ export const Promotions: React.FC = () => {
             title="30K SOV rewards"
```
Comment (bitcoinuser) [inline src/app/pages/LandingPage/components/Promotions/index.tsx:55]: Hi @soulBit, please add this strings to be possible to translate to PT-BR

File: src/app/pages/LandingPage/components/Promotions/index.tsx [hunk 4]
```diff
@@ -55,7 +55,7 @@ export const Promotions: React.FC = () => {
...
             duration="Ongoing weekly rewards"
             text="Provide a 1:1 ratio of SOV and rBTC to the SOV/rBTC AMM liquidity pool and instantly start accruing your share of 30,000 SOV rewards."
```
Comment (bitcoinuser) [inline src/app/pages/LandingPage/components/Promotions/index.tsx:57]: Hi @soulBit, please add this strings to be possible to translate to PT-BR

File: src/app/pages/LandingPage/components/Promotions/index.tsx [hunk 5]
```diff
@@ -55,7 +55,7 @@ export const Promotions: React.FC = () => {
             title="30K SOV rewards"
             duration="Ongoing weekly rewards"
```
Comment (bitcoinuser) [inline src/app/pages/LandingPage/components/Promotions/index.tsx:56]: Hi @soulBit, please add this strings to be possible to translate to PT-BR

File: src/app/pages/LandingPage/components/Promotions/index.tsx [hunk 6]
```diff
@@ -66,7 +66,7 @@ export const Promotions: React.FC = () => {
             title="15K SOV rewards"
             duration="Ongoing weekly rewards"
```
Comment (bitcoinuser) [inline src/app/pages/LandingPage/components/Promotions/index.tsx:67]: Hi @soulBit, please add this strings to be possible to translate to PT-BR

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Please add strings to be able to translate to PT-BR

**Review by Alitasovryn — APPROVED** [review]

**Pull request merged**